app/domain/contactable/email_validator.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+module Contactable
+  class EmailValidator
+
+    def validate_people
+      Person.all.includes(:additional_emails).find_each do |p|
+        if invalid?(p.email)
+          tag_invalid!(p, p.email)
+        end
+        validate_additional_emails(p)
+      end
+    end
+
+    private
+
+    def validate_additional_emails(person)
+      additional_emails = person.additional_emails.pluck(:email)
```

Comment:
Should we extract `person.additional_emails.pluck(:email)` into a separate method, so that wagons can more easily customize which of their email fields are validated? And if so, could we somehow generalize this so `p.email` and additional emails are in the same list and we don't have to treat them separately?